geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/RedisConstants.java

```diff
@@ -45,6 +45,7 @@
   public static final String ERROR_NOT_INTEGER = "value is not an integer or out of range";
   public static final String ERROR_OVERFLOW = "increment or decrement would overflow";
   public static final String ERROR_NAN_OR_INFINITY = "increment would produce NaN or Infinity";
+  public static final String ERROR_OPERATION_PRODUCED_NAN = "resulting score is not a number (NaN)";
```

Comment:
Weirdly, it looks like these constants were added as a part of your PR.  Did you rebase with develop?

Reply:
I pulled in the conflicting changes in this file-  didn't do a full rebase.  I'll do that and see if it makes git happier